client/src/components/Auth/Auth.css

```diff
@@ -0,0 +1,28 @@
+.App{
```

Comment:
![image](https://user-images.githubusercontent.com/95619588/228195377-28b60491-ff45-48ea-9faa-62968627dfd4.png)
better do white color for text, think about it in the future
the same and fix width
![image](https://user-images.githubusercontent.com/95619588/228195621-6372e6f2-acd8-4c19-8d58-58ce51dac2f1.png)

Reply:
![image](https://user-images.githubusercontent.com/95619588/228195862-90fbc469-fd5c-4995-aeb8-bc8be362725a.png)